platform/v1alpha1/types.go

```diff
@@ -0,0 +1,54 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	TypeSourced = "Sourced"
+	TypeApplied = "Applied"
+
+	ReasonSourceFailed     = "SourceFailed"
+	ReasonSourceSuccessful = "SourceSuccessful"
+	ReasonApplyFailed      = "ApplyFailed"
+	ReasonApplySuccessful  = "ApplySuccessful"
+)
+
+// PlatformOperatorSpec defines the desired state of PlatformOperator
+type PlatformOperatorSpec struct{}
+
+// PlatformOperatorStatus defines the observed state of PlatformOperator
+type PlatformOperatorStatus struct {
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
Does it make sense to add a `Phase` here?

Reply:
The EP doesn't currently outline the need for this but I think it could be useful. @timflannagan want to weigh in here?

Reply:
What's the use case we had in mind for a dedicated phase field? My perspective is phase-esq fields are largely unneeded, since you can emulate similar logic using status conditions types.

Reply:
Since it sounds like we're not adding a phase field from this discussion I'm going to go ahead and mark this as resolved.